src/styles/_notifications.less

```diff
@@ -8,7 +8,11 @@
 }
 
 .toast-notifications-list-pf {
+  @media(max-width: @screen-xxs-max) {
+    max-width: calc(100% - - 40px); // Set max-width when < 769, so that word-break functions on Firefox & IE.
```

Comment:
Double minus?  @screen-xxs-max = 479, not 769.  Add additional comment on why 40px?